pandas/core/groupby/generic.py

```diff
@@ -908,11 +908,15 @@ class DataFrameGroupBy(GroupBy):
     )
     @Appender(_shared_docs["aggregate"])
     def aggregate(self, func=None, *args, **kwargs):
-
         relabeling = func is None and is_multi_agg_with_relabel(**kwargs)
         if relabeling:
+            # OWO CHANGES
+            import json
```

Comment:
We can explore other tools like np.array().toBytes() and also use the python list_like check thing

Reply:
Depends on if JSON is too slow and if they can't cythonize this code

Reply:
Implemented